src/extension/background-script/connectors/commando.ts

```diff
@@ -0,0 +1,334 @@
+import Hex from "crypto-js/enc-hex";
+import UTF8 from "crypto-js/enc-utf8";
+import LnMessage from "lnmessage";
+import { v4 as uuidv4 } from "uuid";
+
+import Connector, {
+  CheckPaymentArgs,
+  CheckPaymentResponse,
+  ConnectorInvoice,
+  ConnectPeerArgs,
+  ConnectPeerResponse,
+  GetBalanceResponse,
+  GetInfoResponse,
+  GetInvoicesResponse,
+  KeysendArgs,
+  MakeInvoiceArgs,
+  MakeInvoiceResponse,
+  SendPaymentArgs,
+  SendPaymentResponse,
+  SignMessageArgs,
+  SignMessageResponse,
+} from "./connector.interface";
+
+interface Config {
+  host: string;
+  port: number;
+  rune: string;
+  pubkey: string;
+  wsProxy: string;
+  privateKey: string;
+}
+
+type CommandoGetInfoResponse = {
+  alias: string;
+  id: string;
+  color: string;
+};
+type CommandoSignMessageResponse = {
+  zbase: string;
+};
+type CommandoMakeInvoiceResponse = {
+  bolt11: string;
+  payment_hash: string;
+  payment_secret: string;
+};
+type CommandoChannel = {
+  peer_id: string;
+  channel_sat: number;
+  amount_msat: number;
+  funding_txid: string;
+  funding_output: number;
+  connected: boolean;
+  state: string;
+};
+type CommandoListFundsResponse = {
+  channels: CommandoChannel[];
+};
+type CommandoListInvoicesResponse = {
+  invoices: CommandoInvoice[];
+};
+type CommandoPayInvoiceResponse = {
+  payment_preimage: string;
+  payment_hash: string;
+  amount_msat: string;
+  amount_sent_msat: string;
+};
+type CommandoListInvoiceResponse = {
+  invoices: CommandoInvoice[];
+};
+
+type CommandoInvoice = {
+  label: string;
+  status: string;
+  description: string;
+  amount_received_msat: number;
+  bolt11: string;
+  payment_preimage: string;
+  paid_at: number;
+  payment_hash: string;
+};
+export default class Commando implements Connector {
+  config: Config;
+  ln: LnMessage;
+
+  constructor(config: Config) {
+    this.config = config;
+    this.ln = new LnMessage({
+      remoteNodePublicKey: this.config.pubkey,
+      wsProxy: this.config.wsProxy || "wss://lnwsproxy.regtest.getalby.com",
```

Comment:
guess we should not default to the regtest one here, but rather the production one.

Reply:
I think we can even delete this default, it's already being set here to production: https://github.com/getAlby/lightning-browser-extension/blob/feature/commando-connector/src/app/screens/connectors/ConnectCommando/index.tsx#L26

Reply:
+1